ansible/app_options_files/empty

```diff
@@ -0,0 +1 @@
+
```

Comment:
@melekes @Greg-Szabo Having different playbooks is probably the way forward. Ethermint and Basecoin will not converge to be exactly the same. Furthermore once we add another zone that configuration will most likely also be different. I am thinking like the bitcoin zone or any other zone.